### DISCUSSION TITLE
In HTML files singleQuote is not working?

### DISCUSSION THREAD

**Discussion by krisimmig** [prettier/prettier#15484] [Q&A] [answered]
Maybe I missed something in the docs about this, but I can`t get prettier to use singe quotes for html element attributes in *.html files. This is also not working in the [prettier playground](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAhAEQPIGEAqAmgAoCiABABYwC2ANgHwA6UqAEvgLIAy5u3AQQDKQgLxMQUCAFoAVgGdyNQtNx0B8oROZRy5dqQGYdevak6l8AvmwEAlIZfEgArjABm0gBzaWp-fgAkvjcpAycAJ7kMMF0pKgA9EEhYX6m5pbWAHICFs4AJnDyYABOAJYADjBl0BLkkLAIMM6RfBBZMKRQ+L66+glshsZpqABGEPkRJunEDGxwdHQQ5ADuECV0+eioAEJ2DOT4lGWKJ+QcPACs5DsQZXRwJRV0AIbwAHSJs2lmQrh2gWI+Gm-hWZSg+QgK3eAHMXuRROR3C4oGBqtByAAKACU5GA5Dh7wAju8Ki55JRMS8SjCXDQmvJcQBfADcBJexIR5AA2gBdNmEuhcgDUUDgK3ImDecBZP1McMxAHJSnBpYqADTkRUAVQE0gAGobDdJCBqtS83BBFdiBS8lfIEPkzYqKi8YXAAG5lcXWuWJP4AoEg-QBwH4cjyEpgZzUGAVeRIBIJFYp2EQCAwh7SF5QF50CLVMDyd6QGgJHN5gtlIvvBR1YSELK4SWkABipH2-v+YemiXGkx0iQu3AYIHVIAgVRqUHkyFA1JKUOI1IQs5QeZWLwis-HoxKLzAAGs4DAhC96dxwXBkO48w7d-ujyehK6wOCYcgYCUXHBxw6aGUn7fr+IDyO+DwAIouBA8BAT+44KAAHkI4FwFBMHXkgt50PeIBEtB8DEIu8bICALzyNIYpwIU+RjiAX4vPc764BANA0C8pF5nQdFgVAmZwAIMBfmUoxuHAxCPJeYo3neIHUPQADqxzwPIr5wEIq5lNUXowBEpFFjuIAej+gQQk0QilJUMACBCQi6Q8Mk4SBFSLg6Cn7hUpEuUUjwete47gg6JQwERbrsY5uGuiUQWkfJ3Hji54IwApZT5DAlDIF4AAM44lHA+FlHloUwuFWGyeOMAvKMKVpRlSAAEzjuScD4FVa7YbhcA0KM1E0dwOa0m6cCtus7FCe+nGWiATJMkAA). 

Is this expected behaviour? Are the attributes not supposed to be singe quotable for some reason?

**Comment by seiyab** [comment, marked as answer]
I found that it is discussed in this issue https://github.com/prettier/prettier/issues/5828.
Summarizing the discussion:
- It had been implemented as an intended behavior for consistency with JSX, that only allows double quotes. https://github.com/prettier/prettier/issues/5828#issuecomment-519371278
- A member said Prettier should respect configuration. https://github.com/prettier/prettier/issues/5828#issuecomment-537513833
- Another member attempted to respect singleQuote in HTML. It's not done. https://github.com/prettier/prettier/pull/14562

(I didn't know [single quotes are valid for HTML attributes](https://www.w3.org/TR/html4/intro/sgmltut.html#h-3.2.2). TIL.)

**Comment by sosukesuzuki** [comment]
@seiyab Thank you for telling us the information. I think we should make progress of https://github.com/prettier/prettier/pull/14562.
